### ISSUE TITLE
Show/hide columns

### ISSUE THREAD

**Issue by hammer** [hammerlab/cycledash#747]

**Comment by ihodes** [comment]
Definitely willfix, can't believe we didn't have an issue on this already.

**Comment by hammer** [comment]
Yeah I'd be interested in seeing some examples of how other large tables handle this feature. Would be nice to couple to user accounts so that we can remember settings across runs. Would also be nice to have "saved views" similar to "saved queries" but that's a lot of state to maintain and navigate...

**Comment by ihodes** [comment]
Definitely all good things. Saved views, or at least remembering your view, seems quite important for this feature to be useful. Done easily enough with local storage or user accounts on a per-run basis.

**Comment by hammer** [comment]
Another crazy thought is moving to a cards-based UI for variants, given the amount of metadata attached to each of them.

**Comment by ihodes** [comment]
I think we should try having all (non-essential) columns collapsed by default. I have a mockup for how this will look, will update.

**Comment by ryan-williams** [comment]
I made some toggle-columns functionality in Spree if that is of interest:

![](http://f.cl.ly/items/0Z3c3w3U3K0L2C09311V/Screen%20Recording%202015-10-15%20at%2007.49%20PM.gif)

I have a whole `<Table>` react component in there with columns that can render themselves in jsx etc. so there's a bunch there, jfwiw.
